src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
In this case, the correct semantic solution would be to use the `img` tag. 
Also logo in most cases is a link to the main page.

```suggestion
      <a href="/" class="header__logo">
          <img src="./logoImgPath" alt="logo" />
       </a>
```